### ISSUE TITLE
Issue - You have defined a form action, but haven't added the FormPolicy to your policy ensemble.

### ISSUE THREAD

**Issue by parvbhullar** [RasaHQ/rasa#4592]
Adding form in domain file showing an issue.
"You have defined a form action, but haven't added the FormPolicy to your policy ensemble."
Rasa 1.2.9 
Am I doing anything wrong??

config.yml
`language: en

pipeline:
  - name: "SpacyNLP"
  - name: "SpacyTokenizer"
  - name: "SpacyFeaturizer"
  - name: "SklearnIntentClassifier"
  - name: "CRFEntityExtractor"
    features: [ ["low", "title", "upper"],
      [
          "bias",
          "low",
          "prefix5",
          "prefix2",
          "suffix5",
          "suffix3",
          "suffix2",
          "upper",
          "title",
          "digit",
          "pattern",
         "ner_features",
      ],
     ["low", "title", "upper"]]

policies:
- epochs: 50
  max_history: 6
  name: KerasPolicy
- max_history: 6
  name: AugmentedMemoizationPolicy
- core_threshold: 0.3
  name: TwoStageFallbackPolicy
  nlu_threshold: 0.8
- name: FormPolicy
- name: MappingPolicy`

`raceback (most recent call last):
  File "/usr/local/bin/rasa", line 10, in <module>
    sys.exit(main())
  File "/Users/parvbhullar/Library/Python/3.7/lib/python/site-packages/rasa/__main__.py", line 76, in main
    cmdline_arguments.func(cmdline_arguments)
  File "/Users/parvbhullar/Library/Python/3.7/lib/python/site-packages/rasa/cli/train.py", line 77, in train
    kwargs=extract_additional_arguments(args),
  File "/Users/parvbhullar/Library/Python/3.7/lib/python/site-packages/rasa/train.py", line 40, in train
    kwargs=kwargs,
  File "uvloop/loop.pyx", line 1417, in uvloop.loop.Loop.run_until_complete
  File "/Users/parvbhullar/Library/Python/3.7/lib/python/site-packages/rasa/train.py", line 87, in train_async
    kwargs,
  File "/Users/parvbhullar/Library/Python/3.7/lib/python/site-packages/rasa/train.py", line 169, in _train_async_internal
    kwargs=kwargs,
  File "/Users/parvbhullar/Library/Python/3.7/lib/python/site-packages/rasa/train.py", line 203, in _do_training
    kwargs=kwargs,
  File "/Users/parvbhullar/Library/Python/3.7/lib/python/site-packages/rasa/train.py", line 331, in _train_core_with_validated_data
    kwargs=kwargs,
  File "/Users/parvbhullar/Library/Python/3.7/lib/python/site-packages/rasa/core/train.py", line 32, in train
    from rasa.core.agent import Agent
  File "/Users/parvbhullar/Library/Python/3.7/lib/python/site-packages/rasa/core/agent.py", line 16, in <module>
    from rasa.core.channels.channel import (
  File "/Users/parvbhullar/Library/Python/3.7/lib/python/site-packages/rasa/core/channels/__init__.py", line 25, in <module>
    from rasa.core.channels.telegram import TelegramInput  # nopep8
  File "/Users/parvbhullar/Library/Python/3.7/lib/python/site-packages/rasa/core/channels/telegram.py", line 4, in <module>
    from telegram import (
  File "/usr/local/lib/python3.7/site-packages/telegram/__init__.py", line 46, in <module>
    from .files.file import File
  File "/usr/local/lib/python3.7/site-packages/telegram/files/file.py", line 23, in <module>
    from future.backports.urllib import parse as urllib_parse
  File "/Users/parvbhullar/Library/Python/3.7/lib/python/site-packages/future/backports/__init__.py", line 14, in <module>
    import_top_level_modules()
  File "/Users/parvbhullar/Library/Python/3.7/lib/python/site-packages/future/standard_library/__init__.py", line 810, in import_top_level_modules
    with exclude_local_folder_imports(*TOP_LEVEL_MODULES):
  File "/Users/parvbhullar/Library/Python/3.7/lib/python/site-packages/future/standard_library/__init__.py", line 781, in __enter__
    module = __import__(m, level=0)
  File "/Drives/Masters India/Projects/merlin/nlu_core/test.py", line 3, in <module>
    agent = Agent('domain.yml')
  File "/Users/parvbhullar/Library/Python/3.7/lib/python/site-packages/rasa/core/agent.py", line 305, in __init__
    "You have defined a form action, but haven't added the "
rasa.core.domain.InvalidDomain: You have defined a form action, but haven't added the FormPolicy to your policy ensemble.
`

**Comment by sara-tagger** [comment]
Thanks for the question, our Github issue board is for **bugs and feature requests only,** though 😕
Please ask this question in the [forum✨](https://forum.rasa.com/c/rasa)

###### If you were describing a bug, please create a [report](https://github.com/RasaHQ/rasa/issues/new?labels=bug&template=bug_report.md) and fill out the template
